src/Sarif.Converters/ToolFormat.cs

```diff
@@ -38,5 +38,8 @@ public static class ToolFormat
 
         /// <summary>Static Driver Verifier's file format.</summary>
         public const string StaticDriverVerifier = nameof(StaticDriverVerifier);
+
+        /// <summary>PyLint file format.</summary>
+        public const string PyLint = nameof(PyLint);
```

Comment:
Put this into alphabetical order. NOTE: I made a mistake when I added `TSLint`. While you're here, would you please put that one in the right order as well? Thanks.